### PR TITLE
Fix/del rot smid rmid tests

### PR DESCRIPTION
**Pull request by 2byrds**
Catches KERIA up with KERIpy changes AND provides all functionality necessary for SIGNIFY-TS integration tests to pass, especially multisig-vlei-issuance but also fixes for rotation, salty, and delegation

### CODE REVIEW

File: src/keria/app/delegating.py
Comment (m00sey) [inline src/keria/app/delegating.py:122]: duplicated comment?

**Comment by 2byrds** [comment]
There is a small amount of overlap with https://github.com/WebOfTrust/signify-ts/pull/234/files but we'll adjust once either is merged.

File: src/keria/app/aiding.py
Comment (pfeairheller) [inline src/keria/app/aiding.py:486]: Please delete this code if its no longer needed.

File: src/keria/app/delegating.py
Comment (pfeairheller) [inline src/keria/app/delegating.py:230]: If this method is no longer a part of the class, this annotation is inappropriate

File: src/keria/app/aiding.py
Comment (pfeairheller) [inline src/keria/app/aiding.py:1918]: Are there really cases where the contents of smids could be either a dict or a str?  We should figure out why and if that's necessary

File: src/keria/app/aiding.py
Comment (pfeairheller) [inline src/keria/app/aiding.py:1924]: Same as above

**Review by pfeairheller — CHANGES_REQUESTED** [review]

**Comment by 2byrds** [comment]
@pfeairheller I have made all the updates based on your review. I also added a small change to grouping.py to address @daviddm [issue](https://github.com/WebOfTrust/signify-ts/issues/227) and corresponding PR which contains an additional signify-ts test (https://github.com/WebOfTrust/signify-ts/pull/258) that will now pass and can be merged after this.

I also ran against all original singify-ts integration tests again:
image

**Review by pfeairheller — APPROVED** [review]

**Pull request merged**
